docs(stack): reorganize navigation into evaluate and deployment paths (#1605)
Consensys/doc.linea: docs/stack/how-it-works/core-components.mdx

---
title: Deployment components
description: Required and optional components in a Lineth deployment footprint
sidebar_position: 1
image: /img/socialCards/protocol-components.jpg
---

> Use this page to identify which Lineth (formerly the Linea Stack) components an operator must or may run, and how they are versioned or replaced. For what each component does internally, see the [Protocol architecture](../../protocol/architecture/index.mdx).

Reviewer: ```suggestion
This page describes the Lineth (formerly the Linea Stack) components.
It identifies which components an operator must or may run, and describes how they are versioned or replaced. For what each component does internally, see the [Protocol architecture](../../protocol/architecture/index.mdx).
```
